spec/intl.html

```diff
@@ -1340,6 +1340,270 @@ <h1>CalendarEraYear ( _calendar_, _dateLike_ )</h1>
             1. Return _result_.
           </emu-alg>
         </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatetoiso" type="abstract operation">
+          <h1>
+            CalendarDateToISO (
+              _calendar_: a String,
+              _fields_: a Record,
+              _overflow_: an Object,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It performs implementation-defined processing to convert _fields_ that describes a date in the calendar represented by the string _calendar_ to the ISO calendar and returns the corresponding day, month and year values as a Record with the fields [[Day]], [[Month]] and [[Year]] respectively.</dd>
+          </dl>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateaddition" type="abstract operation">
+          <h1>
+            CalendarDateAddition (
+              _calendar_: a String,
+              _date_: a %Temporal.PlainDate%,
+              _duration_: a %Temporal.Duration%,
```

Comment:
```suggestion
              _duration_: a Duration Record,
```
(it comes from BalanceDuration)